File: DisplaBackend/DisplaBackend/Models/ArticuloVario.cs
using System;
using System.Collections.Generic;

namespace DisplaBackend.Models
{
    public partial class ArticuloVario
    {
        public ArticuloVario()
        {
            ComprobanteItem = new HashSet<ComprobanteItem>();
            PrecioArticulo = new HashSet<PrecioArticulo>();
            VentaVirtual = new HashSet<VentaVirtual>();
        }

        public int Id { get; set; }
        public string Nombre { get; set; }
        public int StockMinimo { get; set; }
        public decimal StockActual { get; set; }
        public decimal PrecioCosto { get; set; }
        public double PorcentajeUtilidad { get; set; }
        public int IdTipoArticulo { get; set; }
        public bool Borrado { get; set; }

        public virtual TipoArticulo IdTipoArticuloNavigation { get; set; }
        public virtual ICollection<ComprobanteItem> ComprobanteItem { get; set; }
        public virtual ICollection<PrecioArticulo> PrecioArticulo { get; set; }
        public virtual ICollection<VentaVirtual> VentaVirtual { get; set; }
    }
}

File: DisplaFrontend/src/app/model/articuloVario.ts
import { TipoArticulo } from './tipoArticulo';
import { PrecioArticulo } from './precioArticulo';

export interface ArticuloVario {
    Id: number,
    Nombre: string,
    StockMinimo: number,
    StockActual: number,
    PrecioCosto: number,
    PorcentajeUtilidad: number,
    IdTipoArticulo: number,
    IdTipoArticuloNavigation: TipoArticulo,
    Borrado: boolean,
    PrecioArticulo: PrecioArticulo[]
}